src/scss/_variables.scss

```
@@ -62,6 +62,14 @@ $color-background-dimmed: rgba(0, 0, 0, 0.5) !default;
  * Color for ad-related UI elements
  */
 $color-highlight-ads: #ffc737 !default;
+/**
+ * Color for ad-related icons
```

Since this color doesn't apply to all icons during ad playback, I'd explicitly mention this.

```suggestion
 * Color for specific ad-related icons such as the skip button icon
```